request.js

```diff
@@ -630,6 +625,11 @@ Request.prototype.init = function (options) {
     }
   }
 
+  // Auth must happen last in case signing is dependent on other headers
```

Comment:
This comment should be moved back to the place where it was. It is related to all of the auth schemes not only oauth. Here you can write a short single line comment saying that oauth's body_hash might need the body.